addons/project_issue/project_dashboard.xml

```diff
@@ -8,7 +8,7 @@
                 <field name="use_issues"/>
                 <field name="issue_ids" invisible="1"/>
             </field>
-            <xpath expr="//div[contains(@class, 'o_project_kanban_boxes')]" position="inside">
+            <xpath expr="//a[contains(@class, 'o_first')]" position="after">
```

Comment:
Why not trying to find the first span.o_label.first() instead of adding a class ?